### PR TITLE
docs: Story aggregation and overview page

### PR DESCRIPTION
**Pull request by dmisdm**
# Objective
This adds a new story that contains all other (relevant) stories on a single page, allowing people to click+drag+zoom around, very similar to figma/sketch.

# Motivation and Context
This is a nice-to-have, and should be a big time saver for when we need to verify that theme switching works across all Kaizen components, because everything is on a single page.

# Screenshots (if appropriate)

https://user-images.githubusercontent.com/7651503/108005466-e95ddc00-704c-11eb-8a07-a28d5dfa5177.mov

# Checklist

- [ ] If this contains visual changes, has it been reviewed by a designer?
- [ ] If this introduces either a new component or a breaking change to an existing component, has it been reviewed by an advocate? (Ask in #prod_design_systems)
- [ ] I have considered likely risks of these changes and got someone else to QA as appropriate
- [ ] I have or will communicate these changes to the front end practice

# Additional Considerations
- Does there need to be right-to-left (RTL) options for localization and internationalization?
- Has the test suite been updated?
- Have you done cross-browser testing for our [supported browsers](https://academy.cultureamp.com/hc/en-us/articles/204539569-Supported-browsers-for-Participants)?
- Have you updated any relevant documentation or left useful comments in the code?
- Have you reviewed Culture Amp's Web Accessibility guide [Current Compliance, Going Forward, Statement and Answers for Customers](https://cultureamp.atlassian.net/wiki/spaces/Prod/pages/428572998/Web+Accessibility)?
- If this contains substantial visual changes, especially far reaching ones, have you unlocked the Chromatic step in the pipeline?
- Have Storybook stories been updated?

**If you're new to Kaizen, please ask #prod_design_systems to set up an onboarding session to get you up to speed.** If you have an urgent PR to merge before that happens, it is safest to ask in #prod_design_systems to have a design system advocate review the PR to catch any issues.

### CODE REVIEW

File: packages/design-tokens/index.stories.tsx [hunk 1]
```diff
@@ -1,56 +1,452 @@
-import { Box } from "@kaizen/component-library"
-import { Divider } from "@kaizen/draft-divider"
-import { Meta } from "@storybook/react"
+/* Stories Modules */
```
Comment (dmisdm) [inline packages/design-tokens/index.stories.tsx:1]: This is 100% not the cleanest code in the world, but didn't want to spend too much time on it.
We can either merge this and come back to it, or just not merge it yet.

**Comment by ActuallyACat** [comment]
I'm not quite sure if we should be running visual regression tests on this story. I took a peek [in chromatic](https://www.chromatic.com/test?appId=5e12c297b9d9020020789ca3&id=602afb4d894fdc0023fa3a02) and most of the document is cut off. Perhaps change the story so Chromatic ignores it? https://www.chromatic.com/docs/ignoring-elements

**Comment by dmisdm** [comment]
> I'm not quite sure if we should be running visual regression tests on this story. I took a peek [in chromatic](https://www.chromatic.com/test?appId=5e12c297b9d9020020789ca3&id=602afb4d894fdc0023fa3a02) and most of the document is cut off. Perhaps change the story so Chromatic ignores it? https://www.chromatic.com/docs/ignoring-elements

Oh nice thanks TIL!

File: packages/design-tokens/index.stories.tsx [hunk 2]
```diff
@@ -1,56 +1,465 @@
...
+    ref: React.Ref<HTMLDivElement>
+  ) => {
+    const theme = useTheme()
+    const border = `solid 1px ${theme.color.wisteria[700]}`
+    return (
+      <div
+        ref={ref}
+        style={{
+          borderLeft: border,
+          borderTop: border,
```
Comment (ActuallyACat) [inline packages/design-tokens/index.stories.tsx:168]: This top/left border is a nice touch. Very Heart. Nice.

**Review by ActuallyACat — APPROVED** [review]

**Pull request merged**